### PR TITLE
Nm mob fixes

### PR DESCRIPTION
**Pull request by dallano**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Implemented and or updated:
Aiatar to have a PH set in order for him to spawn as well as set appropriate HP values.
Hakutaku to use the proper spell list and appropriate level / HP values.
King Vinegarroon to properly despawn when weather changed to normal weather if he wasn't claimed. Implemented his regen to be more powerful at night, and implemented his draw in.
Minotaur to use draw in on the entire alliance appropriately.
Picolaton to have the appropriate drop tables and HP value, as well as set up his placeholder ID

## Steps to test these changes
1. Aiatar: Can use !spawnmob 16900306 to test that he is spawning appropriately. Or by killing his placeholders.

2. Hakutaku: Allow him to cast his spell list to determine the spell list is correct. And kill to determine HP of 50,000 is correct.

3. King Vinegarroon: Run away during battle to trigger draw in. Die after weather has changed from earth weather to determine he despawns, or attempt to spawn him during incorrect weather. Gauge his regen during night vs. day.

4. Minotaur: Run away during battle to trigger draw in.

5. Picolaton: Kill until thrakon breastplate drops to determine drop list is correct.

### CODE REVIEW

File: scripts/zones/Den_of_Rancor/mobs/Hakutaku.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.GIL_MIN, 1600)
+    mob:setMobMod(xi.mobMod.GIL_MAX, 3500)
+    mob:setMod(xi.mod.TRIPLE_ATTACK, 45)
+end
+
+entity.onMobDeath = function(mob, player, isKiller)
+end
+
+return entity
```
Comment (Xaver-DaRed) [inline scripts/zones/Den_of_Rancor/mobs/Hakutaku.lua]: Add newline at the end of the file

File: scripts/zones/Riverne-Site_A01/mobs/Aiatar.lua [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+-- Area: Riverne - Site A01
+--  Mob: Aiatar
+-----------------------------------
+require("scripts/globals/mobs")
+-----------------------------------
+local entity = {}
+
+entity.onMobDeath = function(mob, player, isKiller)
+    UpdateNMSpawnPoint(mob:getID())
+end
```
Comment (Xaver-DaRed) [inline scripts/zones/Riverne-Site_A01/mobs/Aiatar.lua]: Add newline at the end of the file

File: scripts/zones/Western_Altepa_Desert/mobs/Picolaton.lua [hunk 3]
```diff
@@ -2,6 +2,7 @@
 -- Area: Western Altepa Desert
 --   NM: Picolaton
 -----------------------------------
+require("scripts/globals/status")
```
Comment (Xaver-DaRed) [inline scripts/zones/Western_Altepa_Desert/mobs/Picolaton.lua]: This require shouldn't be needed.

File: scripts/zones/Riverne-Site_A01/mobs/Flamedrake.lua [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+local entity = {}
+
+entity.mobOnDeath = function(mob, player, isKiller)
+end
+
+entity.onMobDespawn = function(mob)
+    xi.mob.phOnDespawn(mob, ID.mob.AIATAR_PH, 10, 75600) -- 50 minutes
+end
+
+return entity
```
Comment (Xaver-DaRed) [inline scripts/zones/Riverne-Site_A01/mobs/Flamedrake.lua]: Add newline at the end of the file

**Comment by dallano** [comment]
Great! Thanks for the quick feedback. Fixed the issues. Will be avoided in the future easily.

File: sql/mob_droplist.sql [hunk 5]
```diff
@@ -17438,6 +17438,7 @@ INSERT INTO `mob_droplist` VALUES (3223,0,0,1000,16017,@COMMON);
 INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2811,@VCOMMON);
 INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2815,@COMMON);
 INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2819,@VCOMMON);
+INSERT INTO `mob_droplist` VALUES (3225,0,0,1000,11343,@COMMON); --(Picolaton) Thrakon Breastplate
```
Comment (TeoTwawki) [inline sql/mob_droplist.sql:17441]: sql comments need a space after the dashes or it doesn't import proper on all platforms. 
`--`  vs `-- `

File: sql/mob_spell_lists.sql [hunk 6]
```diff
@@ -4212,6 +4212,19 @@ INSERT INTO `mob_spell_lists` VALUES('Ixzdei_BLM', 467, 176, 1, 255);
 INSERT INTO `mob_spell_lists` VALUES('Ixzdei_BLM', 467, 181, 1, 255);
 INSERT INTO `mob_spell_lists` VALUES('Ixzdei_BLM', 467, 186, 1, 255);
 INSERT INTO `mob_spell_lists` VALUES('Ixzdei_BLM', 467, 191, 1, 255);
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,249,1,255); -- Blaze Spikes
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,235,1,255); -- Burn
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,100,1,255); -- Enfire
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,147,1,255); -- Fire IV
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,148,1,255); -- Fire V
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,176,1,255); -- Firaga III
+INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',468,204,1,255); -- Flare
```
Comment (TeoTwawki) [inline sql/mob_spell_lists.sql]: 468 is already used by `Ixzdei_RDM`. also, the large whitespace gap isn't desired.

File: scripts/zones/Western_Altepa_Desert/mobs/Phorusrhacos.lua [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+require("scripts/globals/mobs")
+-----------------------------------
+local entity = {}
+
+entity.onMobDeath = function(mob, player, isKiller)
+end
+
+entity.onMobDespawn = function(mob)
+    xi.mob.phOnDespawn(mob, ID.mob.PICOLATON_PH, 10, 6400)
+end
```
Comment (Xaver-DaRed) [inline scripts/zones/Western_Altepa_Desert/mobs/Phorusrhacos.lua:16]: missing `return entity`

**Comment by dallano** [comment]
Should be good to go! Thanks for your guys' feedback. This is my first time helping out with private servers so I'm learning lots!

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**